sympy/core/tests/test_wester.py

```diff
@@ -1223,7 +1223,7 @@ def test_N12():
 
 def test_N13():
     x = Symbol('x')
-    assert solveset(sin(x) < 2, domain=S.Reals) == S.Reals
+    assert solveset(sin(x) < 2, domain=S.Reals) == Interval(0, 2*pi, False, True)
```

Comment:
As of now, your result is incorrect and you should not implicitly restrict the domain of the function. When you have ability to represent infinite unions or intersections then you can make the other required changes.